packages/manager/modules/enterprise-cloud-database/src/service/maintenance-window/maintenance-window.constants.js

```diff
@@ -0,0 +1,26 @@
+export const WEEKDAY_NUMBERS = [{
+  day: 'Monday',
+  number: 1,
+}, {
+  day: 'Tuesday',
+  number: 2,
+}, {
+  day: 'Wednesday',
+  number: 3,
+}, {
+  day: 'Thursday',
+  number: 4,
+}, {
+  day: 'Friday',
+  number: 5,
+}, {
+  day: 'Saturday',
+  number: 6,
+}, {
+  day: 'Sunday',
+  number: 7,
```

Comment:
These could probably be fetched from `moment`

Reply:
This has been removed from the constants